code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -624,8 +624,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			var/obj/item/organ/internal/eyes/eye_organ = species_human.get_organ_slot(ORGAN_SLOT_EYES)
 			if(eye_organ)
 				eye_organ.refresh(call_update = FALSE)
-				for(var/mutable_appearance/eye_overlay in eye_organ.generate_body_overlay(species_human))
-					eye_overlay.pixel_y += height_offset
+				for(var/eye_overlay in eye_organ.generate_body_overlay(species_human))
 					standing += eye_overlay
```

Comment:
This can just be `standing += eye_organ.generate_body_overlay(species_human)` now right?